### PR TITLE
Terraform 0.14 upgrade

### PR DESCRIPTION
**Pull request by maximmi**
## what
- Upgrade to support Terraform 0.14 and bring up to current Cloud Posse standard

## why
- Support Terraform 0.14

### CODE REVIEW

File: examples/complete/fixtures.us-east-2.tfvars [hunk 1]
```diff
@@ -1,8 +1,8 @@
 enabled = true
 
-region = "us-east-2"
+region = "us-west-2"
```
Comment (danjbh) [inline examples/complete/fixtures.us-east-2.tfvars:3]: @maximmi Should we rename this file? And I'm guessing we'd need to update the automated tests as well if we do.

**Review by osterman — APPROVED** [review]

**Pull request merged**